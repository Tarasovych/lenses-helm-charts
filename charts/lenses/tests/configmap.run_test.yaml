suite: test default values in configmap
templates:
  - configmap.yaml
tests:
  - it: should work
    asserts:
      - isKind:
          of: ConfigMap
      - isEmpty:
          path: data."lenses\.append\.conf"
