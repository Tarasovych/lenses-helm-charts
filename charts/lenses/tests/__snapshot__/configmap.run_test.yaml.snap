should work:
  1: |
    apiVersion: v1
    data:
      lenses.append.conf: ""
    kind: ConfigMap
    metadata:
      labels:
        app: RELEASE-NAME
        chart: lenses-4.2.6
        heritage: Helm
        lenses.io/app: RELEASE-NAME
        lenses.io/app.type: lenses-configmap
        release: RELEASE-NAME
      name: RELEASE-NAME
